frontend/public/components/routes/create-route.tsx

```diff
@@ -541,57 +555,64 @@ export class CreateRoute extends React.Component<{}, CreateRouteState> {
                     />
                   </div>
                   <div className="form-group co-create-route__insecure-traffic">
-                    <label htmlFor="insecure-traffic">Insecure Traffic</label>
+                    <label htmlFor="insecure-traffic">{t('create-route~Insecure Traffic')}</label>
                     <Dropdown
                       items={
                         termination === 'passthrough'
                           ? passthroughInsecureTrafficTypes
                           : insecureTrafficTypes
                       }
-                      title="Select insecure traffic type"
+                      title={t('create-route~Select insecure traffic type')}
                       dropDownClassName="dropdown--full-width"
                       id="insecure-traffic"
                       onChange={this.changeInsecureTraffic}
                       describedBy="insecure-traffic-help"
                     />
                     <div className="help-block" id="insecure-traffic-help">
-                      <p>Policy for traffic on insecure schemes like HTTP.</p>
+                      <p>{t('create-route~Policy for traffic on insecure schemes like HTTP.')}</p>
                     </div>
                   </div>
                   {termination && termination !== 'passthrough' && (
                     <>
-                      <h2 className="h3">Certificates</h2>
+                      <h2 className="h3">{t('create-route~Certificates')}</h2>
                       <div className="help-block">
                         <p>
-                          TLS certificates for edge and re-encrypt termination. If not specified,
-                          the router&apos;s default certificate is used.
+                          {t(
+                            'create-route~TLS certificates for edge and re-encrypt termination. If not specified, the router&apos;s default certificate is used.',
+                          )}
                         </p>
                       </div>
                       <div className="form-group co-create-route__certificate">
                         <DroppableFileInput
                           onChange={this.onCertificateChange}
                           inputFileData={this.state.certificate}
                           id="certificate"
-                          label="Certificate"
-                          inputFieldHelpText="The PEM format certificate. Upload file by dragging &amp; dropping, selecting it, or pasting from the clipboard."
+                          label={t('create-route~Certificate')}
+                          inputFieldHelpText={t(
+                            'create-route~The PEM format certificate. Upload file by dragging &amp; dropping, selecting it, or pasting from the clipboard.',
```

Comment:
Remove HTML entity

Reply:
The HTML entity &amp; is the escaped string of &.
Changed the HTML entity &amp; to & in file.
Thanks.